portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About</title>
+    <link rel="stylesheet" href="about.css">
+    <script src="truths_and_lies.js"></script>
+  </head>
+  <body>
+    <div class="about_content">
+    <div id="header">
+        <a href="index.html"> <img src=/images/profile.jpg /> </a>
+        <div class="left">
+            <a href="#"> <p> About </p> </a>
+            <a href="survey.html"> <p> Survey </p> </a>
+            <a href="contact.html"> <p> Contact </p> </a>
+        </div>
+    </div>
+    <div id="about">
+        <h3>Want to get to know me? Let's play Two Truths One Lie</h3>
+        <div id="truth_and_lies_game">
+            <label id="question_num">Press Start to Begin!</label>
+            <div id="questions">
+
+                <!-- These fields will be populated by the about.js file -->
+
+                <div class="truth_and_lies_question">
+                    <label>
+                        <input type="radio" name="chosen_answer" id="first_answer" value="0">
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="first_img">
+                    </label>
+                    <p id="first_txt"></p>
+                </div>
+                <div class="truth_and_lies_question">
+                    <label>
+                        <input type="radio" name="chosen_answer" id="second_answer" value="1"/>
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="second_img">
+                    </label>
+                    <p id="second_txt"></p>
+                </div>
+                <div class="truth_and_lies_question">
+                    <label>
+                        <input type="radio" name="chosen_answer" id="third_answer" value="2"/>
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="third_img">
+                    </label>
+                    <p id="third_txt"></p>
+                </div>
+            </div>
+            <button id="two_truths" type="submit" onClick=next()>Start</button>
+            <button id="tutorial" onclick="open_tutorial()">How to Play</button>
```

Comment:
I'd give this an id of tutorial-button.  Really not sure if there is a Google policy on this, but if I'm reading the CSS for this and see that there is a definition for the style of "#tutorial", I'm going to almost assume that it's like a div or something that contains a tutorial.

Also, for method and variable naming in javascript, the style guide says to use camelCase.  So this would be openTutorial()
https://google.github.io/styleguide/jsguide.html#naming-method-names
=)

Reply:
Change tutorial to tutorial-button and open-tutorial to openTutorial()